### PR TITLE
initial pass at non-cookbook supermarket install section

### PR DESCRIPTION
**Pull request by brewn**
Signed-off-by: brewn <nbrewer@chef.io>

### CODE REVIEW

**Comment by brewn** [comment]
#61

File: chef_master/source/install_supermarket.rst [hunk 1]
```diff
@@ -124,7 +124,7 @@ All of the keys under ``node['supermarket_omnibus']`` are written out as ``/etc/
 
 .. code-block:: ruby
 
-   default['supermarket_omnibus']['chef_server_url'] = 'https://chefserver.mycompany.com'
+   default['supermarket_omnibus']['chef_server_url'] = 'https://chefserver.mycompany.com:443'
```
Comment (robbkidd) [inline chef_master/source/install_supermarket.rst:127]: Not sure why this port would need to be specified as that's the default for `https://`. Is it to trigger in the reader "OH, we run SSL on a different port than that!" thoughts?

File: chef_master/source/install_supermarket.rst [hunk 2]
```diff
@@ -291,6 +291,57 @@ When the bootstrap operation is finished, do the following:
...
+
+     .. code-block:: bash
+
+        rpm -Uvh /path/to/package/supermarket*.rpm
+
+#. Run the ``reconfigure`` command to complete the initial installation:
+
+   .. code-block:: none
+
+      sudo supermarket-ctl reconfigure
```
Comment (robbkidd) [inline chef_master/source/install_supermarket.rst:321]: Why this initial reconfigure before creating `/etc/supermarket/supermarket.json`?

File: chef_master/source/install_supermarket.rst [hunk 3]
```diff
@@ -291,6 +291,57 @@ When the bootstrap operation is finished, do the following:
...
+
+#. Create an ``/etc/supermarket/supermarket.json`` file and add the following information, substituting the values for each configuration option with the OAuth 2.0 client credentials that were created in the `previous section </install_supermarket.html#chef-identity>`__:
+
+   .. code-block:: ruby
+
+      {
+          "chef_server_url": "https://chefserver.mycompany.com:443",
+          "chef_oauth2_app_id": "0bad0f2eb04e935718e081fb71asdfec3681c81acb9968a8e1e32451d08b",
+          "chef_oauth2_secret": "17cf1141cc971a10ce307611beda7ffadstr4f1bc98d9f9ca76b9b127879",
+          "chef_oauth2_verify_ssl": false
```
Comment (robbkidd) [inline chef_master/source/install_supermarket.rst]: This list leaves out `chef_identity_url` which is necessary to set in the latest stable version but that will quickly be fixed by what's coming in a release soon.

We should also add setting `fqdn`to these steps for when the private Supermarket is to serve itself up as a name that's different than the hostname of the server.

**Review by robbkidd — CHANGES_REQUESTED** [review]

File: chef_master/source/install_supermarket.rst (hunk 2 above)
Comment (brewn) [reply chef_master/source/install_supermarket.rst:321]: This initial reconfigure creates the `/etc/supermarket` directory

Comment (robbkidd) [reply chef_master/source/install_supermarket.rst:321]: OK.

**Pull request merged**